jsk_spotkinova_robot/README.md

```diff
@@ -229,3 +230,33 @@ You can use `inverse-kinematics` to move arm.
 ```
 (send *spotkinova* :head :inverse-kinematics (make-coords :pos #f(700 0 500) :rotation-axis nil))
 ```
+
+### How to use spotkinova's camera
+
+There are some comera types.
+
+1. Spot (ex. /spot/camera/frontleft/image) ;; This image is monochrome.
+2. Kinova (ex. /kinova_wrist_camera/color/image_raw) ;; This image is RGB color, but the initial image is upside down.
+3. 360 degree camera (ex. /dual_fisheye_to_panorama/output_mouse_left) ;; This image is RGB color and 360 degree.
+
+[HSI color filter](https://jsk-docs.readthedocs.io/projects/jsk_recognition/en/latest/jsk_pcl_ros/nodes/hsi_color_filter.html) can be used with spotkinova's camera.
+
+*This process is only for RGB color camera, so spot camera itself cannot use this filter.*
+
+[Coral USB](https://github.com/knorth55/coral_usb_ros.git) is plugged into the Spotkinova's internel PC, so you can use it with spotkinova's camera.
+
+*This process needs ssh*
+
+```bash
+ssh ichikura@belka.local
+[password]
+source /opt/ros/melodic/setup.bash
+source /home/spot/spot_coral_ws/devel/setup.bash
```

Comment:
```
ssh spot@belka.local
[password]
source /opt/ros/melodic/setup.bash
source $HOME/spot_coral_ws/devel/setup.bash
```
?

Reply:
Is there not /home/spot/ directory? I should have copied and paste.